test/roles.spec.js

```diff
@@ -47,13 +54,38 @@ describe('Role', () => {
   it(' should validate the new role created has a unique title.', (done) => {
     request.post('/api/roles').set('x-access-token', adminToken).set('Accept', 'application/json').send({
       title: 'admin'
-    }).expect(200).expect({message: 'Role already exists.'}).end(done);
+    })
+    .expect(200)
+    .expect({ message: 'Role already exists.' })
+    .end(done);
+  });
+
+  it(' should validate the new role created with a unique title was created.', (done) => {
+    request.post('/api/roles').set('x-access-token', adminToken).set('Accept', 'application/json').send({
+      title: 'level 1'
+    })
+    .expect(200)
+    .end((req, res) => {
+      expect(res.body.message).to.equal('Role was created.');
+      done();
+    });
+  });
+
+  it(' should validate the new role cannot be created wiht no title.', (done) => {
+    request.post('/api/roles').set('x-access-token', adminToken).set('Accept', 'application/json').send({
+    })
+    .expect(200)
+    .expect({ message: 'Role title cannot be null.' })
+    .end(done);
   });
 
   it(' should show that a non-admin user cannot create a new role.', (done) => {
     request.post('/api/roles').set('x-access-token', token).set('Accept', 'application/json').send({
       title: 'non-admin'
-    }).expect(401).expect({message: 'Access denied.'}).end(done);
+    })
+    .expect(401)
```

Comment:
### Critical

There's a more appropriate status code for access denied/forbidden